Main

kaiserdave opened the pull request:
Thanks for the list, it's been very useful. These commits are tweaks that I made during my last playthrough, now rebased onto the current master head.

jontheapple commented:
Not sure if it matters that much, since I don't know if the owner of the repo is still around, but you should consider squashing these commits.

jontheapple commented:
On that note, I looked up Saulden's gift. I haven't gotten there in my own playthrough yet, but it looks like it's possible to get both the Ring of Steel Protection and the Soul Vessel from him, under different circumstances: http://darksouls2.wikidot.com/saulden-the-crestfallen-warrior

jontheapple commented:
Also, with removing non-SotFS stuff, if you create a gap in the numbering system, the checkbox tracking breaks. You need to keep it continuous, starting from "1".

On that note, a lot of the "Travel to" stuff that you moved you made start from "0", which means they don't get tracked because the javascript starts tracking from "1".

kaiserdave commented:
Thanks for the feedback, @jontheapple. I'm reviewing again now for a new playthrough, and the tracking is now fixed. I never paid attention to it before, so I originally only bothered to pick unique IDs and didn't notice the break from the 0 starts and missing sequence numbers.

I've also added the other possible gift from Saulden. Good call.

I'll prefer to leave commits unsquashed, especially during review to make changes easier, but squashing is certainly an option. Personal preference, etc.

kaiserdave commented:
Also, I've reordered commits from "more likely to merge" to "less likely".

kaiserdave committed:
Add missing closing tags

kaiserdave committed:
Fix a gesture name in the checklist

kaiserdave committed:
Fix item pickups

kaiserdave committed:
Add Fire Seed pickup outside Aldia's Keep

kaiserdave committed:
Update the Saulden gift item

kaiserdave committed:
Start sections with the relevant "Travel to" steps

It's more useful to have bonfire travel steps at the start of the
appropriate section instead of at the end of the previous section,
especially if completed sections are collapsed and hidden.

kaiserdave committed:
Clarify some directions

jontheapple commented:
I actually forked a repo off of this one and already added your changes. You can find that repo here: https://github.com/jontheapple/dark-souls-2-sotfs-cheat-sheet/commits/master
